### ISSUE TITLE
Build Failure for serde_codegen_internals

### ISSUE THREAD

**Issue by KodrAus** [serde-rs/serde#432]
The build for `serde_codegen_internals` is failing on `nightly rustc 1.12.0-nightly (5e18b4bad 2016-07-08)`:

```
/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_codegen_internals-0.2.0/src/attr.rs:520:5: 550:6 error: method `fold_tt` has an incompatible type for trait:
 expected &-ptr,
    found enum `syntax::tokenstream::TokenTree` [E0053]
/home/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/serde_codegen_internals-0.2.0/src/attr.rs:520     fn fold_tt(&mut self, tt: TokenTree) -> TokenTree {
```

Looks like a pretty simple fix, I can smack out a PR for it if you like.

**Comment by dtolnay** [comment]
This was fixed in #430 and released in [0.3.0](https://crates.io/crates/serde_codegen_internals/0.3.0).

**Comment by KodrAus** [comment]
Ah coolbeans, I'll update my deps :+1:
